go-controller/pkg/ovn/controller/services/repair.go

```diff
@@ -97,6 +121,27 @@ func (r *repair) runBeforeSync() {
 	}
 	klog.V(2).Infof("Deleted %d stale service LBs", len(staleLBs))
 
+	// Delete those stale template vars
+	staleTemplates := staleTemplateNames.UnsortedList()
+	deleteTemplateVar := &nbdb.ChassisTemplateVar{
+		Variables: make(map[string]string, len(staleTemplates)),
+	}
+	for _, templateName := range staleTemplates {
+		deleteTemplateVar.Variables[templateName] = ""
+	}
+	if err := libovsdbops.DeleteChassisTemplateVarValues(r.nbClient, deleteTemplateVar, func(item *nbdb.ChassisTemplateVar) bool {
+		for _, templateName := range staleTemplates {
+			if _, found := item.Variables[templateName]; found {
+				return true
+			}
+		}
+		return false
```

Comment:
Here you would use the `DeleteChassisTemplateVarVariables` I suggested above.

Reply:
Ack.